### PR TITLE
Fix Laravel login error using Laravel auto instrumentation

### PR DESCRIPTION
**Pull request by eliseuborges**
## Why?
After login on larval application with my credentials, the below error is showed. 
`
OpenTelemetry\Contrib\Instrumentation\Laravel\LaravelInstrumentation::OpenTelemetry\Contrib\Instrumentation\Laravel\{closure}(): Argument #3 ($response) must be of type ?Illuminate\Http\Response, Illuminate\Http\RedirectResponse given, called in /var/www/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php on line 155 `

## How I fixed it?
I changed $response on post method to Symfony\Component\HttpFoundation\Response.
It fix https://github.com/open-telemetry/opentelemetry-php/issues/944

### CODE REVIEW

**Review by pdelewski — APPROVED** [review]

**Comment by pdelewski** [comment]
@eliseuborges Thank you for 1 PR, highly appreciated.

**Comment by eliseuborges** [comment]
> @eliseuborges Thank you for first PR, highly appreciated.

Thanks mate!

File: src/Instrumentation/Laravel/src/LaravelInstrumentation.php [hunk 1]
```diff
@@ -54,7 +55,7 @@ public static function register(): void
 
                 return [$request];
             },
-            post: static function (Kernel $kernel, array $params, ?Response $response, ?Throwable $exception) {
+            post: static function (Kernel $kernel, array $params, Response|RedirectResponse|null $response, ?Throwable $exception) {
```
Comment (brettmc) [inline src/Instrumentation/Laravel/src/LaravelInstrumentation.php:58]: This looks ok, but don't both these responses share a common interface that might be more appropriate?

Comment (pdelewski) [reply src/Instrumentation/Laravel/src/LaravelInstrumentation.php:58]: Good point. It seems that `RedirectResponse` is inherited from `Response`

**Comment by bobstrecansky** [comment]
@eliseuborges are you still planning on working on this PR?

**Comment by eliseuborges** [comment]
Hey Bob. Yes I am. Sorry for my delay.

**Comment by eliseuborges** [comment]
@bobstrecansky and @brettmc thanks for your comments. I fix it!

**Comment by pdelewski** [comment]
@eliseuborges Please fix style issues

```
vendor/bin/php-cs-fixer fix --config=.php-cs-fixer.php -v --using-cache=no -vvv
```

**Comment by eliseuborges** [comment]
Thanks @pdelewski!

**Comment by brettmc** [comment]
@eliseuborges one last thing before we can accept your PR is for you to sign a CLA

**Comment by eliseuborges** [comment]
> @eliseuborges one last thing before we can accept your PR is for you to sign a CLA

Done!

**Pull request merged**